data/world/terrains/desert/beach/init.lua

```diff
@@ -115,7 +115,9 @@
 --        to hide the harsh edges of the triangles. This describes the
 --        `z layer <https://en.wikipedia.org/wiki/Z-order>`_ of a terrain when
 --        rendered next to another terrain. Terrains with a higher value will be
---        dithered on top of terrains with a lower value. Example::
+--        dithered on top of terrains with a lower value. No two terrains may
```

Comment:
No two terrains **should**

Reply:
No, it's mandatory – two identical dither layers are treated as a fatal error during loading